### ISSUE TITLE
The new Spotify update breaks the Custom app "Lyrics" on Synced tab.

### ISSUE THREAD

**Issue by Lainasan** [JulienMaille/dribbblish-dynamic-theme#175]
**Describe the bug**

The new Spotify update breaks the Custom app "Lyrics" on Synced tab and the lyrics is on the top that you can't see it.
Edit: But I can see the Lyrics Normally on other themes.
**To Reproduce**
1. Wait for the next spicetify or spotify update or update the DribbblishDynamic theme itself.

**Screenshots**

![image](https://user-images.githubusercontent.com/90865965/155296410-c3fb6fd5-0db4-40de-9970-d0d1a9fd19e1.png)

**Desktop Setup**
- Operating System: Windows 11 Pro build 22557.1
- Spotify: v1.1.79.763
- Spicetify: v2.9.1
- Dribbblish: v4.1.1-0e50a92

**Logs**

```console
xpui.js:1 Failed to resolve endpoints via x-resolve. Using fallbacks! Error: x-resolve request timed out after 3000 ms
    at xpui.js:1:431268
    at async H._endpointsProvider (xpui.js:1:431731)
(anonymous) @ xpui.js:1
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/vendor~xpui.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/xpui.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/696.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/xpui-desktop-modals.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED

**Comment by Edick1337** [comment]
everything works for me, maybe try reinstalling spotify and spicetify?

**Comment by Lainasan** [comment]
I just uninstall and reinstall spotify and spicetify and apply the theme but the lyrics is still on the top of the tabs.
![image](https://user-images.githubusercontent.com/90865965/155468591-79a2e326-3fd1-481c-a7da-f7a1b65b5369.png)

**Comment by Lainasan** [comment]
Update: The Custom app "Lyrics" is now fixed after the theme update (v4.1.2).
